content/applications/hr/payroll.rst

```diff
@@ -74,17 +71,9 @@ form:
 - :guilabel:`Code`: This code appears with the work entry type on timesheets and payslips. Since the
   code is used in conjunction with the *Accounting* application, it is advised to check with the
   accounting department for a code to use.
-- :guilabel:`Sequence`: The sequence determines the order that the work entry is computed in the
-  payslip list.
-- Check boxes: If any of the items in the list applies to the work entry, check off the box by
-  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
-  This field has a drop-down menu to select the specific type of time off, or a new type of time off
-  can be entered.
-
-.. image:: payroll/new-work-entry.png
-   :align: center
-   :alt: New work entry type form.
-
+- :guilabel:`DMFA Code`: this code is used to identify DMFA entries on a corresponding DMFA report,
+  and is for Belgian-based companies only.
+- :guilabel:`Color`: select a color for the particular work entry type.
```

Comment:
Hi Tim- Jess caught these because per a review from ZST in another doc (one of hers), he prefers lower-case letters after a colon when making lists like these. I thought we caught all of them, but I'll check to be sure they have all been changed. But that is the reason behind this formatting change. I don't think you could easily see it, but in the thread above you can see Jess made the suggestions for those changes. Since this is a personal preference, and it could be capitalized or be lower-case, since we saw that they had been corrected in other docs to be lower-case, that's why we made these changes.